internal/modifier/cdi.go

```diff
@@ -165,6 +184,8 @@ func filterAutomaticDevices(devices []string) []string {
 
 func newAutomaticCDISpecModifier(logger logger.Interface, cfg *config.Config, devices []string) (oci.SpecModifier, error) {
 	logger.Debugf("Generating in-memory CDI specs for devices %v", devices)
+	// TODO: We should try to load the kernel modules and create the device nodes here.
```

Comment:
We can edit this TODO now, after https://github.com/NVIDIA/nvidia-container-toolkit/pull/976#issuecomment-2866044359